### DISCUSSION TITLE
Copy Extractors

### DISCUSSION THREAD

**Discussion by nthoman** [pnp/syntex-samples#4] [Ideas]
I have multiple document classifications that really use the same extractor. Every time I have to re-train the extractor phrases. It would be great if I could just use my existing one as the starting template and then I can just change what I need to change if anything.

**Comment by jameseccles** [comment]
Thanks for the suggestion! We currently have the ability to save an explanation as a template, but not an entire extractor.

**Reply by nthoman** [reply]
No problem... Keep up the good work!
